### PR TITLE
ADD ERC1155 strategies

### PR DESCRIPTION
**Pull request by dave4506**
Changes proposed in this pull request:
Adds a `erc1155-balance-of` strategy that checks balance of a specified `tokenId`
Adds a `erc1155-balance-of-cv` strategy that is `sqrt` of `erc1155-balance-of`

### CODE REVIEW

**Comment by dave4506** [comment]
Would like to also add (in the future) a multi tokenId strategy that aggregates multiple `erc1155-balance-of` is there a best way to do that?

**Review by samuveth — APPROVED** [review]
Looks good!

```
Strategy: "erc1155-balance-of-cv"
Example query
[
  {
    '0x0B7056e2D9064f2ec8647F1ae556BAcc06da6Db4': 3.1622776601683795,
    '0xcc5Ddc8CCD5B1E90Bc42F998ec864Ead0090A12B': 0,
    '0x0154d25120ed20a516fe43991702e7463c5a6f6e': 17.26267650163207
  }
]
getScores: 2.422s
```

```
Strategy: "erc1155-balance-of"
Example query
[
  {
    '0x0B7056e2D9064f2ec8647F1ae556BAcc06da6Db4': 10,
    '0xcc5Ddc8CCD5B1E90Bc42F998ec864Ead0090A12B': 0,
    '0x0154d25120ed20a516fe43991702e7463c5a6f6e': 298
  }
]
getScores: 4.327s
```

**Comment by dave4506** [comment]
looping back can this be merged?

**Comment by bonustrack** [comment]
@dave4506 Hi, can you remove the yarn.lock file? We don't use Yarn on this repository.

**Comment by dave4506** [comment]
Just updated without `yarn.lock` thanks for the catch.

File: src/strategies/erc1155-balance-of-cv/index.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { strategy as erc1155BalanceOfStrategy } from '../erc1155-balance-of';
+
+export const author = 'bonustrack';
```
Comment (bonustrack) [inline src/strategies/erc1155-balance-of-cv/index.ts:3]:
```suggestion
export const author = 'dave4506';
```

**Review by bonustrack — APPROVED** [review]
Looks good!

**Comment by bonustrack** [comment]
@dave4506 The strategy is live! Sorry that it took that long.

**Pull request merged**